core/admin/static_test.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package admin
+
+import (
+	"testing"
+
+	"github.com/google/keytransparency/integration/fake"
+)
+
+func TestStatic(t *testing.T) {
+	admin := NewStatic()
+	client := fake.NewFakeTrillianClient()
+
+	for _, tc := range []struct {
+		logID     int64
+		add, want bool
+	}{
+		{logID: 0, add: false, want: false},
```

Comment:
I think the names make it slightly easier to read in this case. 
Agreed that they're not strictly necessary in tests.